web/src/RootSwitcher.tsx

```diff
@@ -21,14 +21,15 @@ import buildConfig from './constants/buildConfig'
 import { cmsApiBaseUrl } from './constants/urls'
 import useWindowDimensions from './hooks/useWindowDimensions'
 import { cityContentPattern, createPath, RoutePatterns } from './routes'
+import retryImport from './utils/retryImport'
 
 type PropsType = {
   setContentLanguage: (languageCode: string) => void
 }
 
-const MainDisclaimerPage = React.lazy(() => import('./routes/MainDisclaimerPage'))
-const LandingPage = React.lazy(() => import('./routes/LandingPage'))
-const NotFoundPage = React.lazy(() => import('./routes/NotFoundPage'))
+const MainDisclaimerPage = React.lazy(() => retryImport(() => import('./routes/MainDisclaimerPage')))
+const LandingPage = React.lazy(() => retryImport(() => import('./routes/LandingPage')))
+const NotFoundPage = React.lazy(() => retryImport(() => import('./routes/NotFoundPage')))
```

Comment:
Why only for these three pages?

Reply:
we only have these three lazy imports or did i miss sth. Other import should be included in the main chunk and should not fail

Reply:
Have a look at `CityContentSwitcher`

Reply:
just added retry fkt to `CityContentSwitcher`